account_maturity_date_default/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This addon defines as due date of the journal item the date of the journal entry (only if you have not defined one).
```

Comment:
Remember to split texts on 80 cols, and this misses the explanation about only affecting receivable and payable accounts.